File: src/main/java/com/freya02/bot/versioning/LibraryType.java
package com.freya02.bot.versioning;

public enum LibraryType {
	JDA4("JDA 4"),
	JDA5("JDA 5"),
	BOT_COMMANDS("BotCommands");

	private final String displayString;

	LibraryType(String displayString) {
		this.displayString = displayString;
	}

	public String getDisplayString() {
		return displayString;
	}
}

File: src/main/kotlin/com/freya02/bot/versioning/LibraryType.kt
package com.freya02.bot.versioning

import com.freya02.bot.utils.Utils.isBCGuild
import net.dv8tion.jda.api.entities.Guild

enum class LibraryType(val displayString: String, val githubOwnerName: String, val githubRepoName: String) {
    JDA("JDA", "discord-jda", "JDA"),
    JDA_KTX("JDA-KTX", "MinnDevelopment", "jda-ktx"),
    BOT_COMMANDS("BotCommands", "freya022", "BotCommands"),
    LAVA_PLAYER("LavaPlayer", "lavalink-devs", "lavaplayer"),
    ;

    companion object {
        fun getDefaultLibrary(guild: Guild): LibraryType {
            return if (guild.isBCGuild()) BOT_COMMANDS else JDA
        }
    }
}
